OCPBUGS-2665: Added notice for configuring macvlan (#53535)
openshift/openshift-docs: modules/nw-multus-macvlan-object.adoc

|`master`
|`string`

> |The host network interface to associate with the newly created macvlan interface. If a value is not specified, then it defaults to the default route interface.

Reviewer: ```suggestion
|The host network interface to associate with the newly created macvlan interface. If a value is not specified, then the default route interface is used.
```

|====

[NOTE]
====

> If you specify the `master` key for the plug-in configuration, do not specify the same physical network interface associated with your primary network plug-in to avoid possible conflicts.

Reviewer: ```suggestion
If you specify the `master` key for the plug-in configuration, use a different physical network interface than the one that is associated with your primary network plug-in to avoid possible conflicts.
```
